Corona-Warn-App/src/test/java/de/rki/coronawarnapp/submission/ExposureKeyHistoryCalculationsTest.kt

```diff
@@ -10,66 +11,54 @@ class ExposureKeyHistoryCalculationsTest {
 
     private lateinit var instance: ExposureKeyHistoryCalculations
     private lateinit var converter: KeyConverter
+    private lateinit var now: DateTime
 
     @Before
     fun setUp() {
         converter = object : KeyConverter {
             override fun toExternalFormat(
                 key: TemporaryExposureKey,
-                riskValue: Int
+                riskValue: Int,
+                daysSinceOnsetOfSymptoms: Int
             ) =
                 KeyExportFormat.TemporaryExposureKey.newBuilder()
-                    .setRollingStartIntervalNumber(riskValue * 10)
+                    .setRollingStartIntervalNumber(key.rollingStartIntervalNumber)
+                    .setTransmissionRiskLevel(riskValue)
+                    .setDaysSinceOnsetOfSymptoms(daysSinceOnsetOfSymptoms)
                     .build()
         }
-        instance = ExposureKeyHistoryCalculations(TransmissionRiskVectorDeterminator(), converter)
+
+        instance = ExposureKeyHistoryCalculations(
+            TransmissionRiskVectorDeterminator(),
+            DaysSinceOnsetOfSymptomsVectorDeterminator(),
+            converter
+        )
+
+        now = DateTime(2012, 10, 15, 0, 0)
```

Comment:
Ask: I would suggest to add additional test cases for `toExternalFormat` where `now` is not pointing to midnight (local time?) but different points in time (in UTC)

Reply:
this actually detected a bug! thank you!!!